### PR TITLE
tools: remove checker

### PR DESCRIPTION
**Pull request by csuzhangxc**
remove introduction about `checker` because it is obsolete.

### CODE REVIEW

**Comment by csuzhangxc** [comment]
@GregoryIan @lilin90 PTAL

File: op-guide/migration-overview.md [hunk 1]
```diff
@@ -16,17 +16,16 @@ category: advanced
 |MySQL|127.0.0.1|3306|root|*|
 |TiDB|127.0.0.1|4000|root|*|
 
-在这个数据迁移过程中，我们会用到下面四个工具:
+在这个数据迁移过程中，我们会用到下面三个工具:
```
Comment (lilin90) [inline op-guide/migration-overview.md]:
```suggestion
在该数据迁移过程中，会用到下面三个工具：
```

File: op-guide/migration-overview.md [hunk 2]
```diff
@@ -16,17 +16,16 @@ category: advanced
...
-- checker 检查 schema 能否被 TiDB 兼容
 - mydumper 从 MySQL 导出数据
 - loader 导入数据到 TiDB
 - syncer 增量同步 MySQL 数据到 TiDB
 
 ## 两种迁移场景
 
-- 第一种场景：只全量导入历史数据 （需要 checker + mydumper + loader）；
-- 第二种场景：全量导入历史数据后，通过增量的方式同步新的数据 （需要 checker + mydumper + loader + syncer）。该场景需要提前开启 binlog 且格式必须为 ROW。
+- 第一种场景：只全量导入历史数据 （需要 mydumper + loader）；
```
Comment (lilin90) [inline op-guide/migration-overview.md]:
```suggestion
- 第一种场景：只全量导入历史数据（需要 mydumper + Loader）；
```

File: op-guide/migration-overview.md [hunk 3]
```diff
@@ -16,17 +16,16 @@ category: advanced
...
+- 第二种场景：全量导入历史数据后，通过增量的方式同步新的数据 （需要 mydumper + loader + syncer）。该场景需要提前开启 binlog 且格式必须为 ROW。
```
Comment (lilin90) [inline op-guide/migration-overview.md]:
```suggestion
- 第二种场景：全量导入历史数据后，通过增量的方式同步新的数据（需要 mydumper + Loader + Syncer）。该场景需要提前开启 binlog 且格式必须为 ROW。
```

**Pull request merged**